data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3839 @@
+			choice
+				`	(Go.)`
+					decline
+				`	"What about my pay, for the tribute?"`
+			label tribute
+			action
+				payment 250000
+			`	He seems a bit surprised. "Oh, right. I had forgotten about that. Here: <payment>."`
+
+
+
+mission "Unfettered: Solifuge 7: Hai Shipyard Intel"
+	invisible
+	landing
+	source "Hai-home"
+	destination "Darkcloak"
+	to offer
+		has "Unfettered: Solifuge 6: Shipyard Upgrade: active"
+	on offer
+		conversation
+			`You land and ask the Hai for the Unfettered tribute, which they then prepare. They don't seem to suspect anything. While they load up your ship, you to go to the shipyard to and act like you're considering buying a new ship, and a big one at that. "Unfortunately", you get lost and end up at the facility where they build the Geocoris and Shield Beetles. There is only so much you can learn by looking, so you start discreetly scanning the place. A Hai engineer seems to notice you and comes your way. "Hi, how may I help you," he asks.`
+			choice
+				`	"Don't mind me, I'm just admiring the shipyard. It's so well... optimized, compared to human standards." (Pretend to move on and go scan another area.)`
+				`	"Actually, I'm here to buy a ship."`
+					goto buyship
+				`	"I've never seen such a big shipyard before. How can you manage it?"`
+					goto questions
+			`	"Oh, that it is! Well, I'll leave you to it then." He seems to be watching you, but eventually loses interest as you move far enough away.`
+				goto finishscan
+			label questions
+			`	"It's quite simple, actually. The key is optimizing the distribution of ship parts relative to installation location..." He continues explaining their process, and you try to remember everything so you can add it to your scans as extra information. When he's done, he asks, "Would you like me to give you a tour? My break is soon anyway, so I could take it early for you." The thought crosses your mind that he could be stalling for time if he's onto you.`
+			choice
+				`	"Please! I'd like a tour of all of it, especially the big ships."`
+				`	"No thanks. I came to retrieve a tribute, and I think I should get going now."`
+					goto suspect
+			`	He gives you a VIP tour of the whole shipyard facility, which is too good to be true. The tour ends at the local militia office, which has some questions about that scanner they find on you...`
+				decline
+			label buyship
+			`	"Oh, is that so? Well, normally I can't sell ships, but I could make an exception for you... But you do understand I cannot sell you a warship." He may mean you'll have to pay more for said ship. Better prepare your bank account! You think he would probably suspect something if you backed out now. Noticing your hesitation, he adds, "There's no issue if you don't have the money right now. I can offer you a loan of the exact price of the ship, with a small... bonus."`
+			choice
```

Comment:
After we select a ship, add a final yes/no `choice` for whether the player wants to actually buy the ship.